doc/nestml_language/neurons_in_nestml.rst

```diff
@@ -10,11 +10,11 @@ Modeling neurons in NESTML
 Writing the NESTML model
 ########################
 
-The top-level element of the model is ``neuron``, followed by a name. All other blocks appear inside of here.
```

Comment:
The `neuron` keyword is also present in compartmental models docs. Could you also fix that?

Reply:
Thanks for noticing that! Fixed.